Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Types/FarePolicy/FarePolicyInterCityDetails.hs

```diff
@@ -24,7 +24,7 @@ data FPInterCityDetailsD (s :: UsageSafety) = FPInterCityDetails
     perExtraMinRate :: HighPrecMoney,
     kmPerPlannedExtraHour :: Kilometers,
     deadKmFare :: HighPrecMoney,
-    perDayMaxHourAllowance :: Hours,
+    perDayMaxHourAllowance :: Minutes,
```

Comment:
@0utkarsh We should create a new field as this won't be backward compatible since we cache fare policy during the ride so for existing rides this value would be in hours but calculation will happen considering it as minutes.